File: unicardsoftwaredevelopment-androidmobileappfortopupandrefund-c0e3674d02c4/app/src/main/java/au/com/unicard/tafenswwallet/ui/topup/TopupViewModel.java
package au.com.unicard.tafenswwallet.ui.topup;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class TopupViewModel extends ViewModel {
    private MutableLiveData<String> mText;

    public TopupViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is top up fragment!");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: TAFEWallet/app/src/main/java/com/yuan/tafewallet/ui/topup/TopupViewModel.kt
package com.yuan.tafewallet.ui.topup

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class TopupViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is Topup Fragment"
    }
    val text: LiveData<String> = _text
}